### ISSUE TITLE
Train Dataset

### ISSUE THREAD

**Issue by Mhdaw** [google-research/timesfm#228]
Hello @rajatsen91 , Do you plan to release the training data used for training version 1 and 2? Or the data generation method used for synthetic data?
I want to try continual pretraining, Also Can you share the hyperparameters(batch size, optimizer, loss,...) used for training the models? for Pytorch version to be specific.
Best regards.
